game.rb

```diff
@@ -0,0 +1,19 @@
+require 'date'
+require './classes/item'
+# create game class
+class Game < Item
+  attr_accessor :name, :multiplayer, :last_played_at
+
+  def initialize(name, multiplayer, last_played_at)
+    super(publish_date, archived)
+    @name = name
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  private
+
+  def can_be_archived?
+    super && last_played_at > 2.years.ago
+  end
+end
```

Comment:
- [ ] The `can_be_archived?` method should return true if parent's method returns true AND if `last_played_at`  is older than 2 years to archive this consider adding the today's date and year as shown below
```
  private

  def can_be_archived?
    super && (Date.today.year - last_played_at.year) > 2
  end
```